include/mata/re2parser.hh

```diff
@@ -22,6 +22,12 @@
 #include <mata/nfa.hh>
 
 namespace Mata {
+
+    /**
+     * Parser from regular expression to automata (currently `Nfa` and `Afa` are supported).
+     *
+     * TODO: Is this even needed? Could we move this to Mata::Parser?
```

Comment:
Yes, we can move it `parser.hh`. The question is whether will it be simplification, since `re2parser.cc` will stay anyway so it may be confusing for someone.

Reply:
It is rather suggestion to have one less namespace.

Reply:
The file for `re2parser` can and will stay. Nevertheless, having all parsers in a single namespace makes sense to me. Let us discuss this in our meeting.

Reply:
Agree, merging namespace means reducing complexity in this case.

Reply:
Exactly. We will decide on meeting.

Reply:
This issue has its own issue now: #218. I consider this conversation to be resolved.